nixos/modules/services/network-filesystems/tahoe.nix

```diff
@@ -290,9 +314,10 @@ in
                 shares.happy = ${toString settings.client.shares.happy}
                 shares.total = ${toString settings.client.shares.total}
 
-                [storage]
-                enabled = ${boolToString settings.storage.enable}
-                reserved_space = ${settings.storage.reservedSpace}
+                ${formatSection { storage = settings.storage.settings // {
+                  enabled = boolToString settings.storage.enable;
+                  reserved_space = settings.storage.reservedSpace;
+                }; } }
```

Comment:
I can imagine a future version of this module where the whole tahoe.cfg is generated by the INI generator.  I didn't want to try to go that far in this PR though - which already does one useful thing (expands the supported storage options).